pkg/utils/utils.go

```diff
@@ -364,29 +364,6 @@ func ModifyCommonCR(YamlString string, cr csmv1.ContainerStorageModule) string {
 	return YamlString
 }
 
-// ModifyCommonCRs - update with common values
-func ModifyCommonCRs(YamlString string, cr csmv1.ApexConnectivityClient) string {
-	if cr.Name != "" {
-		YamlString = strings.ReplaceAll(YamlString, DefaultReleaseName, cr.Name)
-	}
-	if cr.Namespace != "" {
-		YamlString = strings.ReplaceAll(YamlString, DefaultReleaseNamespace, cr.Namespace)
-	}
-	if string(cr.Spec.Client.Common.ImagePullPolicy) != "" {
-		YamlString = strings.ReplaceAll(YamlString, DefaultImagePullPolicy, string(cr.Spec.Client.Common.ImagePullPolicy))
-	}
-	path := ""
-	for _, env := range cr.Spec.Client.Common.Envs {
-		if env.Name == "KUBELET_CONFIG_DIR" {
-			path = env.Value
-			break
-		}
-	}
-	YamlString = strings.ReplaceAll(YamlString, KubeletConfigDir, path)
-
-	return YamlString
-}
```

Comment:
Is this related to ACC only? 'CommonCRs' makes me worry that this functionality could be in use elsewhere.

Reply:
ModifyCommonCRs was only called via acc_controller.go and commonconfig.go GetAccController, both of which are gone, so it's fine to remove this as it is unused dead code.